portal/src/css.d.ts

```diff
@@ -1,7 +1,6 @@
-// Since we define `.postcssrc.json`, we cannot use `.module.` to opt-in CSS module on a per-file basis.
-// .module.scss is just a naming convention.
+// .module.css is just a naming convention.
```

Comment:
Actually the previous comment is outdated. `.module.css` really means opting-in CSS module. It is documented [here](https://parceljs.org/languages/css/#css-modules)